content/docs/specification/content.md

```diff
@@ -43,151 +43,193 @@ Further links about URL encoding:
+**Root Redirect URL**
+
+- Key: **root**
+- Recommended
+- Default: Fallback to **www** or **redirect** config
+- Permitted values: "absolute|relative URL"
+- Example: "to=https://example.com"
+- Example: "to=/example/"
+
+**Simplified Regex**
+
+- Key: **from**
+- Permitted values: "simplified regex"
+- Example: "from=/$2/$1/\$3"
+
+**Regex**
+
+- Key: **re**
+- Permitted values: "regex/ordered regex"
+- Note: "Named regexes can be used to reorder the results. a > b > b1 > b2"
+- Example: "`from=\\?query=(?P<a>[^&]+)\\&more=(?P<b>[^&]+)`"
 
 ---
+
 # Dockerv2 Type
-**Type**  
 
-* Key: **type**
-* Mandatory
-* Permitted values: "dockerv2"
-* Example: "type=dockerv2"
+## Description
 
-**Version**  
+To redirect request from Docker CLI to a specific docker image or a docker registry you can use `dockerv2` type. The difference between a Docker CLI request and a normal request is that Docker CLI sends a couple of request with different paths to fetch some information like image tags, manifests and etc. about the image. To handle these requests we use docker's v2 API to fetch the data for each request.
 
-* Key: **v**
-* Mandatory
-* Permitted values: "txtv0"
-* Example: "v=txtv0"
+You can point to a docker registry or a specific image inside your record. With pointing to a registry you can have vanity urls for all the images on that registry instead of only an image.
 
-**Backend URL**  
+## Record Fields
 
-* Key: **to**
-* Mandatory
-* Permitted values: "absolute registry URL"
-* Note: "For non container traffic it will fallback to *website*, *www* or *redirect* config"
-* Example: "to=https://gcr.io/" *pass through namespaces, container and tag*
-* Example: "to=https://gcr.io/txtdirect/container" *pass through tag*
-* Example: "to=https://gcr.io/txtdirect/container:tag" *force specific tag*
+**Type**
 
-**Website Redirect URL**  
+- Key: **type**
+- Mandatory
+- Permitted values: "dockerv2"
+- Example: "type=dockerv2"
 
-* Key: **website**
-* Recommended
-* Default: Fallback to **www** or **redirect** config
-* Note: "Non container traffic redirect"
-* Permitted values: "absolute|relative URL"
-* Example: "to=https://about.txtdirect.org/docs/"
-* Example: "to=/docs/"
+**Version**
+
+- Key: **v**
+- Mandatory
+- Permitted values: "txtv0"
+- Example: "v=txtv0"
+
+**Backend URL**
+
+- Key: **to**
+- Mandatory
+- Permitted values: "absolute registry URL"
+- Note: "For non container traffic it will fallback to _website_, _www_ or _redirect_ config"
+- Example: "to=https://gcr.io/" _pass through namespaces, container and tag_
+- Example: "to=https://gcr.io/txtdirect/container" _pass through tag_
+- Example: "to=https://gcr.io/txtdirect/container:tag" _force specific tag_
+
+**Website Redirect URL**
+
+- Key: **website**
+- Recommended
+- Default: Fallback to **www** or **redirect** config
+- Note: "Non container traffic redirect"
+- Permitted values: "absolute|relative URL"
+- Example: "to=https://about.txtdirect.org/docs/"
+- Example: "to=/docs/"
```

Comment:
In general does a relative URL make sense here? Wouldn't that fall back again?

Reply:
I think it falls back too, doesn't make sense to use relative URLs for fallback endpoint.

Reply:
Ok let's remove this from the docs at least.